lib/processes/monitorBuffer.cpp

```diff
@@ -30,30 +30,26 @@ void monitorBuffer::main_thread() {
         for (struct Buffer* buf : buffers) {
             double last_arrival = get_last_arrival_time(buf);
             if ((cur_time - last_arrival) > timeout && last_arrival > 1) {
-                ERROR("The buffer %s hasn't received a frame for %f seconds.", buf->buffer_name,
-                      (cur_time - last_arrival));
-                ERROR("Closing kotekan because of system timeout.");
                 for (auto& buf : buffer_container.get_buffer_map()) {
                     print_buffer_status(buf.second);
                 }
                 usleep(50000);
-                raise(SIGINT);
+                FATAL_ERROR("The buffer %s hasn't received a frame for %f seconds.\nClosing kotekan because of system timeout.", buf->buffer_name,
```

Comment:
multiline errors with `\n` don't show up in syslog.  So better to just not use them. 
```suggestion
                FATAL_ERROR("The buffer %s hasn't received a frame for %f seconds.  Closing kotekan because of system timeout.", buf->buffer_name,
```